php-checks/src/test/resources/checks/PHPDeprecatedFunctionUsageCheck.php

```diff
@@ -13,6 +13,17 @@
   setlocale("LC_ALL", "") ;    // NOK {{Use the "LC_ALL" constant instead of a string literal.}}
 //          ^^^^^^^^
 
+  __autoload();                // NOK
+  create_function('', 'echo 42;'); // NOK
+  parse_str($str);             // NOK
+  parse_str($str, $array);     // OK
+  gmp_random(4);               // NOK
+  each($foo);                  // NOK
+  assert();                    // OK
+  assert($foo);                // OK
+  assert("$foo");              // NOK
```

Comment:
@pynicolas It would be nice to assert messages for this and `parse_str`

Reply:
@vilchik-elena done in e4f8457c354a764e73a6b73e7e293df6c9c94f6e